crates/core/src/metadata_store/providers/objstore/object_store_version_repository.rs

```diff
@@ -47,7 +49,7 @@ impl ObjectStoreVersionRepository {
         }
         let prefix = Path::from(url.path());
 
-        let object_store = create_object_store_client(url, &object_store)
+        let object_store = create_object_store_client(url, &object_store, &RetryPolicy::None)
```

Comment:
Should this also be configurable via `MetadataClientKind::ObjectStore` ?

Reply:
This is a deliberate choice - I'll add a comment why.

The metadata client layer has its own retry policy which is already directly configurable, and it would be confusing to have two separate stacked policies. 

I've been doing some testing this morning to confirm that this works adequately - and it works as expected.